cfme/common/vm_views.py

```diff
@@ -15,6 +15,12 @@
     BaseQuadIconEntity, BaseTileIconEntity, JSBaseEntity, BaseNonInteractiveEntitiesView)
 
 
+class ProvisionTab(Tab):
+    """Workaround for testing purposes only"""
```

Comment:
If this is linked to a BZ, can you include the # here?

Reply:
@lkhomenk shared a message thread with me regarding this behavior, no applicable BZ for now.